docs/functions.rst

```diff
@@ -0,0 +1,98 @@
+Functions in the Tax and Transfer System
+========================================
+
```

Comment:
I suggest a short text here, what is actually on that page. Describing that the things below are modules containing the functions for the particular compartment of the tax and transfer system.